blueflood-core/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/RollupHandler.java

```diff
@@ -129,6 +143,8 @@ private rollupsOnReadTimers (Timer timer) {
         List<SearchResult> units = null;
         List<Locator> locators = new ArrayList<Locator>();
 
+        Timer.Context c = timerRorCalcUnits.time();
```

Comment:
So there is already a timer for getRollupByGranularity() method: the ```plotTimers.SPLOT_TIMER.timer.time()``` and ```plotTimers.MPLOT_TIMER.timer.time()```. The ```timerRorCalcUnits.time()``` is a subset of that. Essentially, it looks like plotTimers == timerRorCalcUnits + rollupsOnReadTimer

There's only a small piece of code, a for loop, that is not covered by other timers.

Just throwing the question, do you think it's useful to keep those plotTimers?